pkg/shell/base_index.js

```diff
@@ -757,7 +757,12 @@ var phantom_checkpoint = phantom_checkpoint || function () { };
 
         /* Branding */
         function setup_brand(id, default_title) {
-            var os_release = JSON.parse(window.localStorage['os-release'] || "{}");
+            var os_release = {};
+            try {
+                os_release = JSON.parse(window.localStorage['os-release'] || "{}");
+            } catch (ex) {
+                console.warn("Couldn't parse os-release", ex);
+            }
```

Comment:
I confirmed that `cockpit.format(s, dict)` just returns `s` with all macros replaced with an empty string for an empty `dict`, so this works reasonably well.